apps/admin-gui/src/app/admin/pages/admin-page/admin-ext-sources/admin-ext-sources.component.ts

```diff
@@ -37,15 +34,19 @@ export class AdminExtSourcesComponent implements OnInit {
     this.filterValue = filterValue;
   }
 
-  onLoad(): void {
-    this.extSourceService.loadExtSourcesDefinitions().subscribe(() => {
-      this.notificator.showSuccess(this.loadSuccess);
-      this.refreshTable();
+  refreshTable(): void {
+    this.loading = true;
+    this.extSourceService.loadExtSourcesDefinitions().subscribe({
+      next: () => {
+        this.getExtSources();
+      },
+      error: () => {
+        this.getExtSources();
+      },
```

Comment:
According to the task definition, is `loadExtSourcesDefinitions()` necessary if there is no further action with a returned value? 
```suggestion
    this.getExtSources();
```

Reply:
My question would be if this is the desired way of loading the ext sources from config. Shouldn't the database be automatically filled with the contents of the config ? Or can it be changed during run time ? It not there is no reason to load the config on every refresh.